pkg/sloop/ingress/dbrestore_test.go

```diff
@@ -0,0 +1,50 @@
+package ingress
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
+)
+
+func TestDatabaseRestore(t *testing.T) {
+	tempFile, err := ioutil.TempFile("/tmp", "sloop-test-*")
```

Comment:
We really try not to let unit tests interact with the filesystem.  Using the host fs makes it so you are testing more than just your unit of code, can lead to flakyness, host specific issues and slow test runs.  (Less of an issue in a small project like this, but a big problem in million+ line codebases)  I would consider using afero, or tweaking the code so you can test the part that does not touch the filesystem.

Reply:
In this specific case, there is almost no real business logic in DatabaseRestore of significance.  So simply skipping this unit test is fair game unless you feel it is going to catch some regression.

Reply:
I can just drop the test.

Reply:
Done